app/forms/work_form.rb

```diff
@@ -46,6 +46,7 @@ def persisted?
   validates :license, presence: true, on: :deposit
 
   attribute :work_type, :string
+  before_validation ->(work_form) { work_form.work_type.presence }
```

Comment:
Can you say more about what this is doing? Like, if the block returns a `nil` does that short-circuit validation or something?

Reply:
It just changes a `''` to a `nil` so that code using this form doesn't have to remember to treat `''` as `nil`.

The mapper forgot to do this, which is what caused the bug. This is just preventative of future bugs.

Reply:
@justinlittman Doesn't `#presence` return the value instead of mutating?

Reply:
FWIW, when I comment out the `before_validation` the mapper and form specs still pass.

Reply:
```
hungry-hungry-hippo(dev)> ''.presence
=> nil
hungry-hungry-hippo(dev)> 'foo'.presence
=> "foo"
```

Yes, I would expect the tests to pass. For the sake of defensiveness, I fixed the same problem in 2 places.

Reply:
Oh, you're right. Duh.